Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
> My knowledge doesn't understand the impacts of edit Colors to other branchs.

Other packages you mean. I don't know either, but that's not a problem as long as we have a deprecation system : people will be warn, follow the upgrade procedure and be safe.